src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -90,11 +92,21 @@ export class TransferablesGridComponent implements OnInit, AfterViewInit {
     this.statusService.updateDownloadProgress().subscribe(data => {
       this.zone.run(() => {
         this.generalProgress = data;
+        if (data === 100) {
+          this.downloadInProgress = false;
+        } else {
+          this.downloadInProgress = true;
+        }
```

Comment:
would be simpler to write this as 
```
this.downloadInProgress = data !== 100;
```
Also, we need to get rid of magic numbers in this code, such as `100`, but that can wait for a later cleanup.

Reply:
Thanks, we will refactor this in a later cleanup.